src/gui/presets.cpp

```diff
@@ -939,13 +986,55 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Sega System 1", {
+      DIV_SYSTEM_SMS, 64, 0, 2,
+      DIV_SYSTEM_SMS, 64, 0, 3,
+      0
+    }
+  ));
   cat.systems.push_back(FurnaceGUISysDef(
     "Sega System 32", {
       DIV_SYSTEM_YM2612, 64, 0, 4,
       DIV_SYSTEM_YM2612, 64, 0, 4,
       DIV_SYSTEM_RF5C68, 64, 0, 2,
       0
     }
+  ));
+    cat.systems.push_back(FurnaceGUISysDef(
+    "Sega Hang-On", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_SEGAPCM, 64, 0, 0,
+      0
+    }
+  ));
+    cat.systems.push_back(FurnaceGUISysDef(
+    "SNK Alpha-68K", {
```

Comment:
Alpha-68K arcade board is originally manufactured and used by Alpha denshi, Later used or published game developed from this hardware by SNK.